modin/experimental/core/execution/native/implementations/hdk_on_native/dataframe/dataframe.py

```diff
@@ -82,12 +78,11 @@
 )
 from modin.pandas.utils import check_both_not_none
 
-
 IDX_COL_NAME = ColNameCodec.IDX_COL_NAME
 ROWID_COL_NAME = ColNameCodec.ROWID_COL_NAME
+UNNAMED_IDX_COL_NAME = ColNameCodec.UNNAMED_IDX_COL_NAME
 encode_col_name = ColNameCodec.encode
 decode_col_name = ColNameCodec.decode
-concat_index_names = ColNameCodec.concat_index_names
```

Comment:
Why?

Reply:
It's used only in a single method _union_all().

Reply:
Right, but it would be reasonable to have this alias in the function scope. Or remove it at all.